tests/openapi/test_named_vector_crud.py

```
@@ -117,6 +117,7 @@ def test_create_vector_rejects_zero_size(collection_name):
     response = requests.put(
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)